tests/testthat/test-pmcmc.R

```diff
@@ -0,0 +1,337 @@
+context("pmcmc")
+
+data <- read.csv(system.file("extdata/example.csv", 
+                             package = "sircovid", 
+                             mustWork = TRUE), stringsAsFactors = FALSE)
+
+test_that("pmcmc runs without error", {
+ 
+  n_mcmc <- 10
+  set.seed(1)
+  X <- pmcmc(
+    data = data, 
+    n_mcmc = n_mcmc
+  )
+  
+  
+ expect_is(X, 'list')
+ expect_setequal(names(X), c('inputs', 'results', 'states', 'acceptance_rate', 'ess'))
+ expect_equal(dim(X$results), c(n_mcmc + 1L, 5))
+ expect_equal(dim(X$states), c(n_mcmc + 1L, 238))
```

Comment:
It might be useful to save the run with this seed as `pmcmc.Rds`, load it back in and check that it's identical. See "Parameters are generated as before" in `test-parameters.R` and L47-55 in `test-sampler.R`

Reply:
added check and pushed